e3sm_diags/driver/tropical_subseasonal_driver.py

```diff
@@ -130,10 +131,15 @@ def calculate_spectrum(path, variable, start_year, end_year):
         "dosymmetries": True,
         "rmvLowFrq": True,
     }
-    # TODO the time subsetting and variable derivation should be replaced during cdat revamp
+    # TODO the time subsetting and variable derivation should be replaced during
+    # cdat revamp.
```

Comment:
TODO: Is this comment still relevant?

Reply:
Yes, we should use Dataset class to ingest data, so that time subsetting and variable derivation doesn't need to be treated separately, in recent event testing with EAMxx files, I just realized that we do need to make this tropcial_subseasonal_driver to understand EAMxx variable convention as well.